### PR TITLE
removed jquery from addon code

### LINKED ISSUE #16: Remove JQuery
Hi All - 
I'm trying to remove jquery from Ember Light Table (which is two dependencies upstream). 

Hope you'd all consider this (I've done all the work)

- [x] Remove instances of `Ember.$` see #17 
- [x] Upgrade addon/tests See #18

### PR DESCRIPTION
**Pull request by donaldwasserman**
Fixes #16

### CODE REVIEW

**Comment by donaldwasserman** [comment]
Hey @Gaurav0 - would you mind taking a look at this PR and letting me know your feedback? Thanks!

File: addon/components/resize-detector.js [hunk 1]
```diff
@@ -30,10 +30,9 @@ export default Ember.Component.extend({
     if (this.get('isDestroyed') || this.get('isDestroying')) {
       return;
     }
-    let $el = Ember.$(element);
     this.get('on-resize')({
-      width: $el.width(),
-      height: $el.height()
+      width: element.offsetWidth,
+      height: element.offsetHeight
```
Comment (Gaurav0) [inline addon/components/resize-detector.js]: offsetWidth and offsetHeight are not good substitutes for jquery width and height, especially if the user is watching window or document.

For the most compatible solution, you would have to use parseFloat(window.getComputedStyle(element).width) and that still wouldn't match for the case of window or document, where the calculation by jquery gets really complex.

It is quite likely that users are depending on existing behavior. I'm not sure what to do.

Comment (Gaurav0) [reply addon/components/resize-detector.js]: Here's the exact equivalent:

https://github.com/nefe/You-Dont-Need-jQuery#2.2

File: addon/components/resize-detector.js [hunk 2]
```diff
@@ -6,6 +6,37 @@ const {
...
+ */
+
+function getWidth(el) {
+  let styles = window.getComputedStyle(el);
+  let width = el.offsetWidth;
+  let borderLeftWidth = parseFloat(styles.borderLeftWidth);
+  let borderRightWidth = parseFloat(styles.borderRightWidth);
+  let paddingLeft = parseFloat(styles.paddingLeft);
+  let paddingRight = parseFloat(styles.paddingRight);
+  return height - borderLeftWidth - borderRightWidth - paddingRight - paddingLeft;
```
Comment (alexander-alvarez) [inline addon/components/resize-detector.js]: @donaldwasserman  `height -> width`

Comment (donaldwasserman) [reply addon/components/resize-detector.js]: Thanks - sorry, I got distracted with a few other things. 

@alexander-alvarez - changes coming on both repos.

**Comment by donaldwasserman** [comment]
@Gaurav0 @alexander-alvarez - Do you have any other thoughts about this? Do think this is an acceptable solution moving forward?

**Comment by alexander-alvarez** [comment]
Seems fine to me. I'm of the belief that if works :ship: it.

**Review by Gaurav0 — APPROVED** [review]
LGTM

**Comment by Gaurav0** [comment]
Published as v0.4.0

**Pull request merged**